Bio/KEGG/Compound/__init__.py

```diff
@@ -13,19 +13,21 @@
  - Record - A representation of a KEGG Ligand/Compound.
 """
 
-# other Biopython stuff
 from __future__ import print_function
 
-from Bio.KEGG import _write_kegg
-from Bio.KEGG import _wrap_kegg
+from Bio.KEGG import _default_wrap, _wrap_kegg, _write_kegg
 
 
 # Set up line wrapping rules (see Bio.KEGG._wrap_kegg)
 name_wrap = [0, "",
              (" ", "$", 1, 1),
              ("-", "$", 1, 1)]
-id_wrap = lambda indent: [indent, "", (" ", "", 1, 0)]
-struct_wrap = lambda indent: [indent, "", ("  ", "", 1, 1)]
+id_wrap = _default_wrap
+
+
+def struct_wrap(indent):
```

Comment:
Since it seems to get defined twice, why not put ``struct_wrap`` into the main KEGG file too as ``_struct_wrap``?

Reply:
Yes, agree. Have done so.